presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -354,6 +362,25 @@ std::unique_ptr<common::BytesRange> varcharRangeToFilter(
       nullAllowed);
 }
 
+std::unique_ptr<common::BigintRange> dateRangeToFilter(
+    const protocol::Range& range,
+    bool nullAllowed,
+    const VeloxExprConverter& exprConverter,
+    const TypePtr& type) {
+  bool lowExclusive = range.low.bound == protocol::Bound::ABOVE;
```

Comment:
It would be nice to add a unit test for this conversion. The e2e test added in the PR doesn't cover all the code paths here.

Reply:
I see that `presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp` is a potential place to add unit tests. But I agree that this test suite is incomplete. For example, it does not verify the result for some of the tests.
Can you open an issue to improve this test and add more coverage here?
Since this PR is blocking Parquet Date type usage and has an E2E test for the changes, we can merge this for now.

Reply:
Created https://github.com/prestodb/presto/issues/18942.